hw/ip/prim/README.md

```diff
@@ -310,3 +310,59 @@ fusesoc \
     --mapping partner:prim_mytech:all \    # Select alternate implementation via mappings
     lowrisc:systems:chip_earlgrey_asic
 ```
+### prim_asap7 example
+
+ [ASAP7](https://github.com/The-OpenROAD-Project/asap7) is an open-source standard-cell library.
+ It serves as an example for partners to integrate their own technology specific prim library.
+
+The steps in [creating a tech library](#creating-a-technology-library) were followed to create this prim library.
+
+Each standard-cell instance name is prefixed with a `u_size_only_` such that these instances can be easily identified during synthesis and preserved.
+
+### Important synthesis constraints to keep important redundant constructs
+
+The basic prim instances cannot be removed or merged with other cells through logic optimization or constant propagation as this would remove important side channel countermeasures from the design.
+
+All instantiated basic gates (`buf`, `mux2`, `inv`, `clock_gating`, `and2`, `xor2`, `xnor2`, `flop`) should be instantiated with a name prefix of `u_size_only_` such that preserve attributes can be set during synthesis.
+The syntax to set a preserved attribute varies across tool providers.
+To make sure the right constraints are applied, a simple example design (`prim_sdc_example`) is provided.
+This design can be synthesized and its netlist can be analyzed to guarantee that the right constraints are set and all important cells are preserved.
```

Comment:
to guarantee -> to help finding the right constraints for the setup at hand...

We can't give guarantees at this point :-)